### ISSUE TITLE
Add support for Gradle 8

### ISSUE THREAD

**Issue by quoc-huynh-cosee** [Wes1324/eraser#10]
Currenty we get the following error if we are using Gradle 8.
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':eraser'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file: /Users/huynh/.pub-cache/hosted/pub.dev/eraser-2.0.2/android/build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s
Error: Gradle task assembleBetaDebug failed with exit code 1
```

**Comment by Wes1324** [comment]
Thanks for reporting this @quoc-huynh-cosee.

I had to add a namespace while fixing issue #9. That PR has been merged now and I've deployed version 3.0.0 of the package to pub.dev.

Can you see if the newest version fixes the issue you were seeing please?

**Comment by quoc-huynh-cosee** [comment]
Yeah it is working now. Thank you.
